core-services/secret-mirroring/_mapping.yaml

```diff
@@ -372,6 +372,18 @@ secrets:
   to:
     namespace: test-credentials
     name: cluster-secrets-azure-arcconformance-sa
+- from:
+    namespace: ocm-secrets
+    name: acm-cicd-clusterpool-shared
+  to:
+    namespace: test-credentials
+    name: acm-cicd-clusterpool-shared
+- from:
+    namespace: ocm-secrets
+    name: acm-cicd-clusterpool-shared
+  to:
+    namespace: ci
```

Comment:
Why is it needed for `ci` namespace?

Reply:
https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/#use-a-secret-in-non-step-jobs

Because Prow isn't the end of our CICD pipeline, we need our secrets available in both namespaces so that we can also use them in handcrafted post-submit jobs. If CI Operator was able to generate post-submit jobs, we wouldn't need to have it in ci.